paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -217,6 +222,7 @@ internal class PaymentSheetViewModel @Inject internal constructor(
             stripeIntentValidator.requireValid(stripeIntent)
         }.fold(
             onSuccess = {
+                handle!!.set(SAVE_STRIPE_INTENT, stripeIntent)
```

Comment:
!! should be avoided

Reply:
yeah, just temporary, until I get it working on PaymentSheetViewModel